src/atoms/buttons/Button.js

```diff
@@ -8,34 +8,44 @@ function Button ( props ) {
     children,
     onClick,
     shake,
-    className
+    className,
+    buttonType,
+    disabled
   } = props;
 
   return (
-    <div
+    <button
       className={ classnames(className, styles[type], { [styles.shaking]: shake }) }
```

Comment:
@jmcolella gotcha, thanks - I will make the change for button but fwiw I'm not sure about this implementation. All we're doing with `type` at the moment is adding a new class in the sass file and exposing it as an option to the user. If we use explicit types, every time we want to add a new variation (and each component could have 10-15 variations), we'd have to declare it as a possible prop, add it to the classnames variant map, add the new class to the sass file, and expose to the user. It seems like a lot of unnecessary steps, when the trade off is asking the user to write `type="action"` instead of `action`. might be at least worth discussing before we decide to switch everything over